mypy/main.py

```diff
@@ -160,6 +162,8 @@ def parse_version(v):
                         help="an anti-pattern")
     parser.add_argument('--stats', action='store_true', help="dump stats")
     parser.add_argument('--inferstats', action='store_true', help="dump type inference stats")
+    parser.add_argument('--custom-typeshed', metavar='PATH',
+                        help="use the custom typeshed module at PATH")
```

Comment:
I wonder if the term 'typeshed module' is clear enough, as the term 'module' is overloaded? Maybe use 'typeshed repository'?